index.html

```diff
@@ -168,6 +168,31 @@ <h3>Skills</h3>
         
     </ul>
 </section>
-
+<section>
+    <h3 class="info3">
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </h3>
+    </section>
+    <section class="footer">
+    <form
+    action="https://formspree.io/f/xbjenlov"
+    method="POST">
+        <ul>
+            <li><input type="text" class="name-email" name="name" maxlength="30" placeholder="Full name" required></li>
+             <li> <input type="text" class="name-email" name="email" placeholder="Email address" required></li>
+              <li> <textarea class="message" maxlength="500" id="msg" name="user_message"  placeholder="Hey I’d like to connect to..." required></textarea></li>
+               <li><button type="submit" class="my-resume2">Get in touch</button></li>         
+       </ul>
```

Comment:
- [ ] There are also a few style adjustments that need to be made in your contact form. I have noted them in the images below. So kindly fix those as well :+1: 

| Your Design | The Figma Design |
|--------|--------|
| ![image](https://github.com/AhmedMh0/Mobile-Portfolio/assets/102699958/dee8159c-16e5-438d-8821-5329486197ab) | ![image](https://github.com/AhmedMh0/Mobile-Portfolio/assets/102699958/58b3e3ca-8c36-4c23-a775-c91915f0183b) |